README.md

```diff
@@ -80,11 +80,16 @@ Options:
 
 #### Development
 
+*WARNING* The `Makefile` used in the commands below assume your
+containers can be deleted.  Please work on an isolated machine and read
+the `cleanup` target in the `Makefile` prior to executing.
```

Comment:
Thank you. That's definitely an omission on our part.

We should really be switching to a fig.yml for local development.